build.gradle

```diff
@@ -95,6 +100,20 @@ allprojects { thisProject ->
     if (thisProject == rootProject) {
         artifactoryPublish.publishConfigs()
     }
+
+    // Never build zip distributions (to save space and time)
+    // Only build tar distributions for stand-alone services
+    thisProject.plugins.withType(DistributionPlugin) {
```

Comment:
The `application` plugin applies the `distribution` plugin, but the build scripts for subprojects run *after* the build scripts for the root project, so we need to resort to this callback structure.